dashboard/config/routes.rb

```diff
@@ -13,6 +13,10 @@
 
   patch '/api/v1/user_scripts/:script_id', to: 'api/v1/user_scripts#update'
 
+  patch '/api/v1/user_school_infos/:id/update_last_confirmation_date', to: 'user_school_infos#update_last_confirmation_date'
+
+  patch '/api/v1/user_school_infos/:id/update_end_date_last_seen_school_info_interstitial', to: 'user_school_infos#update_end_date_last_seen_school_info_interstitial'
```

Comment:
@madelynkasula  
Since I am updating two different parameters in two different tables, I added end_date to the method name.  Will it be misleading if only the last_seen_school_info_interstitial is part of the method name?

Reply:
that's a great point. i think it's still okay to change in this case because the two parameters are so tightly tied to each other